tests/include/banner.sh

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Can you remove extra newlines here?

Do you plan to add the banner at the start in zopen_check()?

Reply:
Yes I can remove them. I will do so.  I can add to beginning as well.  My apologies I thought you just wanted to pull them in individually.